web/frontend/src/pages/ballot/components/HintButton.tsx

```diff
@@ -0,0 +1,78 @@
+import { FC, Fragment, useRef } from 'react';
+import { QuestionMarkCircleIcon } from '@heroicons/react/outline/';
+import { Popover, Transition } from '@headlessui/react';
+
+type HintButtonProps = {
+  text: string;
+};
+
+const HintButton: FC<HintButtonProps> = ({ text }) => {
```

Comment:
This component is a good candidate for being a global general-purpose component. In such case, it should be placed in `src/components`.

Reply:
Done.